client/src/components/AreaDetail/index.tsx

```diff
@@ -331,7 +331,9 @@ const AreaDetail = ({properties}:IAreaDetailProps) => {
       </ul>
       <div className={styles.categorization}>
         <div className={styles.priority}>
-          <div className={categoryCircleStyle} />
+          {categorization === 'Community of focus' ?
```

Comment:
This string comparison with English text seems fragile. Will it work for a localized instance?

Reply:
Great catch!

Updated to useState!